minutes/2021-09-07.md

```diff
@@ -0,0 +1,43 @@
+# Parsec Community Meeting (07/09/2021) 📋
+
+Minutes of previous meeting are stored
+[here](https://github.com/parallaxsecond/community/tree/main/minutes).
+
+## Attendance
+
+- Matt Davis
+- Sam Davis
+- Paul Howard
+- Wiktor Kwapisiewicz
+- Marc Meunier
+- Ionut Mihalcea
+
+## Agenda
+
+- Update on the TPM key persistence issue
+- SQLiteKIM, design discussion around what should happen in certain edge cases (e.g. the database
+   exists, but tables are missing etc)
+- Think about migration story of Parsec KIMs.
+
+## Minutes
+
+- Work on a fix for the TPM persistence issue is underway
+   - A first version of the fix was confirmed to work, so the focus now is on preparing the whole
+      stack for upstreaming
+   - Despite no explicit request from users exists, we have chosen to implement a low-effort,
+      low-impact migration scheme. This will fully reside in the TPM provider and will rely on
+      `bincode` to flag issues in deserializing old key info structures. If an old structure is
+      found, a migration to the new format is attempted.
+- Matt presented his proposal for handling the edge cases in the bootstrapping code for the SQLite
+   KIM.
+   - If the database file is missing or if all the tables are missing from it (e.g. because the
+      admin wanted to create the empty file with correct permissions), the tables are created and
+      the service construction continues.
+   - If either some of the tables are missing, an error is thrown and the service bootstrapping
```

Comment:
```suggestion
   - If some of the tables are missing, an error is thrown and the service bootstrapping
```

Reply:
Or alternatively `If either of the tables are missing, an error is thrown and the service bootstrapping`

Reply:
Thanks, I took out the 2nd part but forgot to remove the `either` 😅